src/generated/ghe-217-endpoints.ts

```diff
@@ -1279,14 +1279,6 @@ const Endpoints: EndpointsDefaultsAndDecorations = {
     users: ["GET /search/users"]
   },
   teams: {
-    addMember: [
-      "PUT /teams/:team_id/members/:username",
-      {},
-      {
-        deprecated:
-          "octokit.scim.addMember() is deprecated, see https://developer.github.com/enterprise/2.17/v3/teams/members/#add-team-member"
-      }
-    ],
```

Comment:
It should not remove team endpoint methods. Also ... no idea where the `scim` in `octokit.scim.addMember()` came from in the deprecation message 🤷‍♂